app/views/layouts/crawler.html.erb

```diff
@@ -16,7 +16,9 @@
     <%= theme_lookup("head_tag") %>
     <%= render_google_universal_analytics_code %>
     <%= yield :head %>
-
+    <% if show_browser_update? %>
```

Comment:
I wonder if this should be called `force_browser_update?`

I think it could be confusing at the moment because even if `show_browser_update?` is false, we might still show it in javascript 🤔